src/GCodes/GCodes.cpp

```diff
@@ -2543,6 +2543,9 @@ void GCodes::AbortPrint(GCodeBuffer& gb) noexcept
 	{
 		StopPrint(StopPrintReason::abort);
 	}
+	RetractZProbe(gb, -1);
```

Comment:
Calling macros here (abort.g or the delay probe macros) is in general not safe, because AbortPrint is called from so many different contexts. Some of those even set up a different state to return to. We would need to examine each place that calls AbortPrint, and see what we can do in that context. In some cases we may be able to switch to a new "aborting" state, which can handle running the macros.

Reply:
I fixed this in my latest commit.